src/main/java/uk/gov/hmcts/reform/userprofileapi/service/UserProfileCreator.java

```diff
@@ -85,7 +85,6 @@ private UserProfile persistUserProfileWithAudit(CreateUserProfileData profileDat
             if (profileData.getStatus() != null) {
```

Comment:
if profileData is null this will lead to a NullPointerException.
If you reverse the order:
if(null != profileData.getStatus()) {
    // Never returns a NullPointer evaluates as false instead
    // PLEASE REVISE
}